bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
Then I would prefer we set this to ``12.18.31`` to match what we currently use in production.